Common/Data/Custom/Robintrack/RobintrackHoldings.cs

```diff
@@ -0,0 +1,164 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NodaTime;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace QuantConnect.Data.Custom.Robintrack
+{
+    /// <summary>
+    /// Aggregate unique user stock holdings
+    ///
+    /// Data sourced from Robintrack - https://robintrack.net
+    /// </summary>
+    public class RobintrackHoldings : BaseData
+    {
+        /// <summary>
+        /// Number of unique users holding a given stock
+        /// </summary>
+        public int UsersHolding { get; set; }
+
+        /// <summary>
+        /// Total number of unique holdings across all stocks by users
+        /// </summary>
+        public decimal TotalUniqueHoldings { get; set; }
+
+        /// <summary>
+        /// Percentage of the U.S. equities universe that unique users hold stock for
+        /// </summary>
+        public decimal UniverseHoldingPercent => UsersHolding / TotalUniqueHoldings;
+
+        /// <summary>
+        /// Alias for <see cref="UsersHolding"/>
+        /// </summary>
+        public override decimal Value
+        {
+            get { return UsersHolding; }
+            set { UsersHolding = (int)value; }
```

Comment:
Don't think these properties should have public sets?

Reply:
Without it being `public set`, we lose the ability to initialize the object for tests without reflection.

I am in favor of making this immutable, and also eventually applying that pattern to the rest of the BaseData implementations, though it'll definitely result in breaking changes across the board.